tools/docker/generate_dockerfiles.py

```diff
@@ -39,7 +39,7 @@ def main() -> None:
         f.write(toolchain_full() + regtest("sdbg", "minimal"))
 
     with OutputFile(f"Dockerfile.test_cmake", args.check) as f:
-        f.write(spack_env_toolchain() + regtest_cmake())
+        f.write(spack_env_toolchain() + regtest_cmake(testopts="--mpiexec 'mpiexec pika-bind --'"))
```

Comment:
There are like ~5 levels of escaping for testopts - it's really a pain to get right.
For now you can just add it to `test_regtest_cmake.sh` directly because it's not used for any other test.

Reply:
Yep, I was indeed struggling with that. =) I'll do what you suggest, for the time being.